simple_config

```diff
@@ -10,5 +10,7 @@ path /static StaticRequestHandler {
 # Currently: no default options for handlers
 path /echo EchoRequestHandler {}
 
+path /status StatusHandler {}
```

Comment:
The issue was that there was no way to actually visit the StatusHandler's results page.